localize.d.ts

```diff
@@ -0,0 +1,8 @@
+export function _js(origString: string, ...replacements: string[]): string;
+export function ___p(
+   number: number,
+   singularString: string,
+   pluralString: string,
+   ...replacements: string[]
+): string;
+export function setTranslations(translations: any): void;
```

Comment:
I think this is just `Record<string, string>`
```suggestion
export function setTranslations(translations: Record<string, string>): void;
```